packages/astro/src/integration/create-integration.ts

```diff
@@ -13,11 +14,12 @@ type HotloadAstroClerkIntegrationParams = AstroClerkIntegrationParams & {
   clerkJSUrl?: string;
   clerkJSVariant?: 'headless' | '';
   clerkJSVersion?: string;
+  enableEnvSchema?: boolean;
 };
 
 function createIntegration<Params extends HotloadAstroClerkIntegrationParams>() {
   return (params?: Params): AstroIntegration => {
-    const { proxyUrl, isSatellite, domain, signInUrl, signUpUrl } = params || {};
+    const { proxyUrl, isSatellite, domain, signInUrl, signUpUrl, enableEnvSchema = false } = params || {};
```

Comment:
Since this is primarily for convenience and doesn't affect the core functionality of our integration, should we just enable it by default?

Reply:
Yeah, since it's stable in Astro 5 I'd say let's enable it by default and change our docs/quickstart to use it 👍 I guess it's good to give the ability to turn it off in case someone want's to override it (assuming Astro doesn't merge the schema, dunno)